docs/get-started/json-rpc-commands.mdx

```diff
@@ -0,0 +1,879 @@
+---
+id: json-rpc-commands
+title: JSON RPC Commands
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import {useState} from 'react';
+import "../../src/css/custom.css";
+
+export const JsonRpcTerminal = (props) => {
+  const [value, setValue] = useState("");
+  const { method, params, network } = props;
+  return (
+    <div>
+      <div>
+        {value != "" ? <pre className="json_rpc_terminal">{value}</pre> : null}
+      </div>
+      <div>
+        {value == "" ? (
+          <button
+            className="json_rpc_terminal_button"
+            onClick={() => {
+              fetch(network, {
+                method: "POST",
+                headers: {
+                  Accept: "application/json",
+                  "Content-Type": "application/json",
+                },
+                body: JSON.stringify({
+                  jsonrpc: "2.0",
+                  method: method,
+                  params: params,
+                  id: 1,
+                }),
+              })
+                .then((res) => res.json())
+                .then((response) => {
+                  setValue(JSON.stringify(response));
+                });
+            }}
+          >
+            Run command
+          </button>
+        ) : (
+          <button
+            className="json_rpc_terminal_button"
+            onClick={() => {
+              setValue("");
+            }}
+          >
+            Clear Terminal
+          </button>
+        )}
+      </div>
+    </div>
+  );
+};
+
+## eth
+
+### eth_chainId
+
+Returns the currently configured chain id, a value used in replay-protected transaction signing as introduced by EIP-155.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+* None
+
+<h4><i>Returns:</i></h4>
+
+* <b> QUANTITY </b> - big integer of the current chain id.
+
+<h4><i>Example:</i></h4>
+
+Run command and see live results from our testnet.
```

Comment:
Add "the" in front of "command". This is repeating so change it in other places as well

Reply:
Fixed! 
 [b1173b4](https://github.com/0xPolygon/polygon-sdk-docs/pull/41/commits/b1173b4db2a92b5f3c153e1906ad8ac9e82900bf)